### DISCUSSION TITLE
Error with Jaeger in traefik-mesh-proxy pod logs - GKE

### DISCUSSION THREAD

**Discussion by siddharth201983** [jaegertracing/jaeger#3175] [Q&A]
`time="2021-08-02T12:24:48Z" level=error msg="Tracing jaeger error: failed to flush Jaeger spans to server: UDP connection not yet initialized, an address has not been resolved" tracingProviderName=jaeger`

I am trying to deploy traefik-mesh on GKE. All the pods are showing 'Running' but when checking traefik-mesh-proxy pod logs, getting this error.

Can anyone please suggest to me, where I am doing wrong?

**Comment by jpkrohling** [comment]
It would probably be a better question for the Traefik folks.
